runtime/src/rent_collector.rs

```diff
@@ -35,7 +36,11 @@ impl RentCollector {
     // updates this account's lamports and status and returns
     //  the account rent collected, if any
     //
-    pub fn update(&self, account: &mut Account) -> u64 {
+    pub fn update(&self, address: &Pubkey, account: &mut Account) -> u64 {
+        if *address == incinerator::id() && self.epoch >= incinerator::ACTIVATION_EPOCH {
```

Comment:
Until we reach ACTIVATION_EPOCH, incinerator behaves like a normal account.    Getting the operating mode into the RentCollector would require a lot of contortion, and the only consumer of v1.0 is mainnet-beta so I didn't see a reason to do more here